protocol-designer/src/organisms/AssignLiquidsModal/LiquidCard.tsx

```diff
@@ -72,38 +79,74 @@ export function LiquidCard(props: LiquidCardProps): JSX.Element {
     {}
   )
 
+  const handleClearLiquid = (
+    labwareId: SelectedContainerId,
+    wells: string[]
+  ): void => {
+    if (labwareId != null) {
+      dispatch(
+        removeWellsContents({
+          labwareId,
+          wells,
+        })
+      )
+    } else {
+      console.error('Could not clear selected liquid - no labware ID')
+    }
+  }
+
   return (
     <ListItem type="noActive" flexDirection={DIRECTION_COLUMN} key={name}>
       <Flex
+        flexDirection={DIRECTION_COLUMN}
         padding={SPACING.spacing12}
-        alignItems={ALIGN_CENTER}
-        gridGap={SPACING.spacing16}
+        gridGap={SPACING.spacing4}
       >
-        <LiquidIcon color={color ?? ''} size="medium" />
-        <Flex flexDirection={DIRECTION_COLUMN} width="12.375rem">
-          <StyledText
-            desktopStyle="bodyDefaultSemiBold"
-            css={LINE_CLAMP_TEXT_STYLE(3)}
-          >
-            {name}
-          </StyledText>
-          <StyledText
-            desktopStyle="bodyDefaultRegular"
-            css={LINE_CLAMP_TEXT_STYLE(3)}
+        <Flex alignItems={ALIGN_CENTER} gridGap={SPACING.spacing16}>
+          <LiquidIcon color={color} size="medium" />
+          <Flex flexDirection={DIRECTION_COLUMN} width="12.375rem">
+            <StyledText
+              desktopStyle="bodyDefaultSemiBold"
+              css={LINE_CLAMP_TEXT_STYLE(3)}
+            >
+              {name}
+            </StyledText>
+            <StyledText
+              desktopStyle="bodyDefaultRegular"
+              css={LINE_CLAMP_TEXT_STYLE(3)}
+            >
+              {info.liquidIndex != null
+                ? liquidsWithDescriptions[info.liquidIndex].description
+                : null}
+            </StyledText>
+          </Flex>
+          <Flex
+            cursor={CURSOR_POINTER}
+            onClick={() => {
+              setIsExpanded(!isExpanded)
```

Comment:
your way works but also this way should work too!

```suggestion
              setIsExpanded(prev => !prev)
```